packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -47,17 +77,6 @@ export type GPCPCDArgs = {
   // contain bigints if used for network requests (e.g. ProveAndAdd).  The
   // choice here should be driven by the needs of the Prove screen.
 
-  /**
-   * POD objects to prove about.  These are not revealed by default, but
-   * a redacted version of their entries will become part of the claims of the
-   * resulting proof PCD, as specified by the proof config.
-   *
-   * See {@link GPCProofConfig} and {@link GPCRevealedClaims} for more
-   * information.
-   */
-  pod: PCDArgument<PODPCD>;
```

Comment:
i suggest turning `pod` into `pods: PODPCDRecordArg` so as to not pollute this namespace, rather than what you do below with `& PODPCDRecordArg`

Reply:
That's an interesting idea.  Cleaner at this level, though it'll presumably require some more change to the logic of the prove screen in order to display one "arg" to the user as multiple, each with its own description, validator, etc, since we want each POD to be selectable separately.
Beyond this use case in particular, it would make sense to have a general abstraction for multiple args by name in a group like this, and/or args in an array.  If those existed they'd apply here easily, but they might be more complicated to build from scratch.